crates/protocol/registry/src/lib.rs

```diff
@@ -36,9 +37,36 @@ lazy_static::lazy_static! {
     pub static ref ROLLUP_CONFIGS: HashMap<u64, RollupConfig, DefaultHashBuilder> = _INIT.rollup_configs.clone();
 }
 
+/// Returns all available [Chain] identifiers.
+pub fn chain_idents() -> Vec<String> {
+    CHAINS.chains.iter().map(|c| c.identifier.clone()).collect()
+}
```

Comment:
Is there a reason not to make this a method on `CHAINS`? (same goes for below functions as well)